virt/virt-web-console-overview.adoc

```diff
@@ -1384,7 +1409,7 @@ You can manage MigrationPolicies for your workloads on the MigrationPolicies pag
 |*MigrationPolicies* table
 |List of MigrationPolicies
 
-Click the Options menu {kebab} beside a MigrationPolicy to select *Edit* or *Delete*.
+Click the *Options* menu {kebab} beside a MigrationPolicy to select *Edit* or *Delete*.
```

Comment:
Actually, we do not mark a menu name unless it is actually indicted with a tooltip or similar.

Reply:
Yes, I found those sections as well. I asked Andrea Hoffer to confirm.

Reply:
In the meantime, no need to change. I will resolve this conversation.